tests.py

```diff
@@ -25,6 +25,8 @@ def test_listdir(self):
 
         self.filesystem.resolve('dir0').add_child('subdir', InMemoryDir())
         self.assertEqual(self.storage.listdir('dir0'), [['subdir'], []])
+        self.assertEqual(self.storage.listdir('dir0\\subdir'), [[], []])
```

Comment:
It looks like `dir0\\subdir` is passed through as-is on both Linux and Windows, instead of being normalized back to `dir0/subdir` on Linux.

I suppose you could call this "expected behaviour" and drop this assertion.